Orange/preprocess/impute.py

```diff
@@ -207,6 +206,9 @@ def __call__(self, data):
         column[mask] = predicted
         return column
 
+    def transform(self, c):
+        pass  # abstract in Transformation, never used here
```

Comment:
Wouldn't it be better to `assert False, "abstract in Transformation, never used here"`, so that any erroneous calls are caught immediately??